features/steps/project/source/browse_files.rb

```diff
@@ -106,9 +118,37 @@ class Spinach::Features::ProjectSourceBrowseFiles < Spinach::FeatureSteps
   end
 
   step 'I can see new file page' do
-    expect(page).to have_content "New file"
+    expect(page).to have_content "new file"
     expect(page).to have_content "Commit message"
   end
+  
+  step 'I can see "upload existing one"' do
+    expect(page).to have_content "upload existing one"
+  end
+
+  step 'I can see the "user.feature"' do
+    expect(page).to have_content "user.feature"
+  end
+
+  step 'I click on "upload existing one"' do
+    click_link 'upload existing one'
+  end
+  
+  step 'I click on "Upload file"' do
+    click_button 'Upload file'
+  end
+  
+  step 'I upload "user.feature"' do
+    attach_file(:file_upload, File.join('features', 'user.feature'))
+  end
+
+  step 'I see the new content of the ".gitignore"' do
+    expect(page).not_to have_content old_gitignore_content
```

Comment:
@stanhu thanks the 'new_gitignore_content' was defined in the project, and I attach a new different file, so I can not use 'new_gitignore_content', but I can use not 'old_gitignore_content'.